subset/dot1x/test_dot1x

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -e
 
+freeradius -fxx -l stdout &
```

Comment:
how does this work running this in the background without the while loop?  Seems like it would run this and then just go all the way through to the report without ever waiting for anything?

Reply:
It just starts the freeradius server before calling the python script. The idea is to make sure test_dot1x.py blocks until the authenticator module returns a definitive result and then write to the file. For now, it just goes through since test_dot1x.py writes to the file directly.

Reply:
Trevor, it just starts a RADIUS server. The fail/pass would be handled by the python call. Since it is currently just something that writes directly, I think it should be fine.

Reply:
Added a todo to test_dot1x.py